packages/components/tree/src/tree.stories.ts

```diff
@@ -190,6 +191,20 @@ export default {
   title: 'Navigation/Tree',
   tags: ['draft'],
   excludeStories: ['flatData', 'nestedData'],
+  parameters: {
+    a11y: {
+      config: {
+        rules: [
+          {
+            /** The rule is disabled due to unnecessary Storybook a11y bug.
```

Comment:
Can you fix the spacing in the comment? Multiline comments should be formatted like this:

```
/**
 * BLah blah.
 */